Fabric.Authorization.AccessControl/src/app/services/global/auth.service.ts

```diff
@@ -30,14 +30,15 @@ export class AuthService implements IAuthService {
   clientId: string;
   authority: string;
 
-  constructor(private httpClient: HttpClient, private servicesService: ServicesService) {
+  constructor(private httpClient: HttpClient, private configService: ConfigService) {
     this.clientId = 'fabric-access-control';
   }
 
   initialize(): Promise<any> {
     // Need the initializer to load all the values first
-    // This makes sure we get the correct accessControl endpoint
-    return this.servicesService.getIdentityAndAccessControlUrl().pipe(
+    // This makes sure we get the correct accessControl endpoint, from the configService
+    return forkJoin(this.configService.getIdentityServiceRoot(), this.configService.getAccessControlServiceRoot()).pipe(
```

Comment:
this is ok, except when windows auth is enabled.  If windows auth is enabled, then we do not use the config service to get these 2 urls, we use discovery service

Reply:
The thought here was that these two url's are being passed in during authorization install, and we could use those to setup the auth client

Reply:
If you look at the service, i think I did make it so Auth pushes all values every time.  Might be safe to do this then.

Have you found a way to get around the multiple discovery service calls?

Reply:
This is what was done initially with the previous pr, which was sending an undefined token because the auth.service wasn't built out yet when the discovery request went off.

https://github.com/HealthCatalyst/Fabric.Authorization/blob/18bdefa0ee645ce52518fefa7c956812f84e3734/Fabric.Authorization.AccessControl/src/app/services/interceptors/fabric-http-request-interceptor.service.ts#L42

Running into a similar issue now with the auth.guard not firing off (currently the services list if being initialized in the initializer service which hooks into the angular initialization).